checks.d/snmp.py

```diff
@@ -62,6 +62,9 @@ def __init__(self, name, init_config, agentConfig, instances=None):
         # Create SNMP command generator and aliases
         self.create_command_generator(mibs_path, ignore_nonincreasing_oid)
 
+        # Set OID batch size
+        self.oid_batch_size = int(init_config.get("oid_batch_size", DEFAULT_OID_BATCH_SIZE))
```

Comment:
Not sure what's better, I purposely made this parameter settable at the check level. I can move it to an instance level instead,